cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -60,23 +68,31 @@ def _include_file(self, root_parts, f):
             return lower_f.endswith((".js", ".js-meta.xml", ".html", ".css", ".svg"))
         return True
 
-    def _get_files_to_package(self, path):
+    def _get_files_to_package(self):
         for root, dirs, files in os.walk("."):
             root_parts = root.split(os.sep)[1:]
             if self._include_directory(root_parts):
                 for f in files:
                     if self._include_file(root_parts, f):
                         yield os.path.join(root, f)
 
+    def _get_static_resource_files(self):
+        for root, dirs, files in os.walk("."):
+            root_parts = root.split(os.sep)[1:]
+            for f in files:
+                if self._include_file(root_parts, f):
```

Comment:
I don't think I understand calling `_include_file()` here. Shouldn't we zip the static resource bundle content-blind?

Reply:
Yeah I think this was copy-paste coding. Will remove.